### ISSUE TITLE
local dashboard says "node not connected" [BUG]

### ISSUE THREAD

**Issue by stn021** [bittorrent/go-btfs#293]
**Describe the bug**
Hello, I have installed a btfs-node with 'btfs init' and then 'btfs daemon'
I also set up a new storage path with btfs storage path <path> <volume>

The node appears in the btfs-explorer but it tells me storage = 9.32GB. That is wrong, it should be more
Also the local dashboard shows the correct path but the tabs "dashboard", "cheques" and "files" tell me "Looks like your node is not connected"

**To Reproduce**
btfs init ; btfs daemon
2. See error
in btfs-explorer

**Expected behavior**
see the correct storage-status of my node in the btfs-explorer

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Ubuntu 22.04] ubuntu 22.04
 - Version [e.g. 2.2.1]

**Additional context**

**Comment by FRiceSchoolAcct** [comment]
"Looks like your node is not connected" means the network connectivity is unstable or something is wrong. 
You may check the following:
The "settings" page should be visible; the 'api endpoint' and 'storage path' is correct or not? Have you clicked the 'submit' button?

About the '9.32GB', my host(ubuntu 20.04) is correct. What's the exact storage in your config file?
  "Datastore": {
    "StorageMax": "100GB",
    "StorageGCWatermark": 90,
    "GCPeriod": "1h",
    "Spec": {...

**Comment by stn021** [comment]
> About the '9.32GB', my host(ubuntu 20.04) is correct. What's the exact storage in your config file? "Datastore": { "StorageMax": "100GB", "StorageGCWatermark": 90, "GCPeriod": "1h", "Spec": {...

Well, I set storage-path AND size with the CLI-command
./btfs storage path <absolute path> <size>

The path was set up correctly, I can see that on the disk
The size was NOT set up correctly. It was ignored. I later found out that I have to manually change it in the config-file even thought I had already done that.

This may be a bug, not sure.

**Comment by Richard2GitHub** [comment]
OK, what's the CLI-command to set storage size?  To set storage size, ref: https://docs.btfs.io/docs/change-my-hosting-folder-1#using-command-line

**Comment by stn021** [comment]
I have now repeated the process with what I know so far:

1) ./btfs storage path <new path> <size> # to set path AND size, as written in the docs
2) manually wrote size into file <new path>/config
3) restarted btfs daemon
4) checked on https://scan.btfs.io/#/node/<... id of my node>
5) check the local GUI

results:
- in the explorer the new size is NOT shown, not even after 10 minutes
- in the local GUI
  - the path is correct (page "settings")
  - the size is AFAIK not visible
  - and page "dashboard" say "not connected"

The "not connected" is not a network-problem, it is rented server with direct connection to the internet. I have no idea what might cause this, please advise.

**Comment by Richard2GitHub** [comment]
What's your host id? By the way, have you clicked the 'SUBMIT' button on the 'SETTINGS' page? If not, please have a try, and then you can see the storage size info on the 'DASHBOARD' page.

**Comment by stn021** [comment]
> What's your host id? By the way, have you clicked the 'SUBMIT' button on the 'SETTINGS' page? If not, please have a try, and then you can see the storage size info on the 'DASHBOARD' page.

Okay, I did that and now the dashboard works. The explorer still shows the wrong size (10 minutes later) but that may be just a delay.

To me btfs looks like a server-software that can run on a headless server.
Therefore I find it an unexpected behaviour that I have to click SUBMIT.
Normally I set up any server-software with CLI only and never with any GUI.

The reason here is that I usually put software like btfs on a rented server and use SSH for all interaction. Any kind of GUI is only on occasion and only for checking and never for purposes of configuration.
